Python/ast_unparse.c

```diff
@@ -911,6 +953,15 @@ maybe_init_static_strings(void)
         !(_str_dbl_close_br = PyUnicode_InternFromString("}}"))) {
         return -1;
     }
+    if (!_str_inf &&
+        !(_str_inf = PyUnicode_InternFromString("inf"))) {
+        return -1;
+    }
+    if (!_str_replace_inf &&
+        !(_str_replace_inf = PyUnicode_FromFormat("1e%d", 1 + DBL_MAX_10_EXP))) {
+        return -1;
+    }
+    PyUnicode_InternInPlace(&_str_replace_inf);
```

Comment:
I don't think that it's useful to intern these strings. I suggest to remove PyUnicode_InternInPlace and replace PyUnicode_InternFromString with PyUnicode_FromString.